pymc_bart/tree.py

```diff
@@ -140,7 +140,11 @@ def new_tree(
     ) -> "Tree":
         return cls(
             tree_structure={
-                0: Node.new_leaf_node(value=leaf_node_value, idx_data_points=idx_data_points)
+                0: Node.new_leaf_node(
+                    value=leaf_node_value,
+                    nvalue=len(idx_data_points),
```

Comment:
I think mypy complains that can be `None` son `len` can not be immediately applied. I think you need to add the `if idx_data_points is not None` clause.